docs/modules/ROOT/pages/How to Access Payara Micro Binaries.adoc

```diff
@@ -0,0 +1,68 @@
+= How to Access Payara Micro Enterprise Binaries for Local Development
+
+Payara Cloud customers with a Premium subscription have access to Payara Micro Enterprise binaries to test their applications locally on the same environment they will use when they deploy to the cloud.
+
+As part of the Payara Cloud Premium subscription, you are entitled to: 
+
+* Full access to the Payara Customer support portal for raising support tickets for 2 users 
+* A single, shared Nexus repository account for your organisation 
+
+__Sonatype Nexus__is a repository manager which we use to provide our Enterprise builds and private builds for Payara Micro. It also contains the binaries for our connectors (Cloud, Testing, Security, etc …), Docker Images, and IDE plugins which are tested with our Enterprise Edition software. The repository can also be used to store large files for your support tickets. 
+
+By the time you receive this guide, accounts should have been created for you on these two key platforms. 
+
+[[verify-your-customer-support-portal-access]]
+== Verify Your Customer Support Portal Access
+
+You should have received an email with a link to create a password for your support portal 
+
+account. Please click the link and set a password to be able to view your tickets on the web. 
+
+If you would like to add/swap named contacts for your company, you may request to do so either through a support ticket or by emailing mailto:support@payara.fish[+++support@payara.fish+++. ]The maximum number of Zendesk users for Payara Cloud support is 2. 
+
+*Change Your Nexus Password*
+
+You should receive an email with your Nexus credentials, which contains a default password. Since email notifications are an inherently insecure communication method, it is your responsibility to change the password immediately. 
```

Comment:
These sentences run off to the right of the screen instead of staying within the margins. Have to scroll to the right to read them and then scroll back.

Reply:
This seems just a graphical glitch with netlifly - I don't know how to fix that

Reply:
It doesn't occur locally